src/lib/receptions/sequencescapeUtils.js

```diff
@@ -246,7 +255,13 @@ const buildPool = ({ labware, included, barcodeAttribute, libraryOptions }) => {
  * @param {Object} requestOptions Additional request parameters, will over-ride any
  * @returns {Object} tubes_attributes object ready for import into traction
  */
-const transformTube = ({ labware, included, requestOptions, barcodeAttribute }) => {
+const transformTube = ({
```

Comment:
Looks like transformTube and transformPlate have the same arguments. Maybe worth a type?

Reply:
Done.